ui/src/containers/WidgetVisualiseCreator/index.js

```diff
@@ -84,9 +84,9 @@ class WidgetVisualiseCreator extends Component {
             schema={schema}
             id={this.getVisualisationId()}
             displayCount={3}
-            parseOption={model => createDefaultTitleWithIcon(model, model.get('description'))}
-            parseOptionString={model => model.get('description', createDefaultTitle(model))}
```

Comment:
When model.get('description') is empty string `""`, `parseOptionString(model)` will not be a string but an object. That causes displaying `[object Object]` in widget visualisation `SimpleInput`

Reply:
Nice catch 👍